bin/install

```diff
@@ -14,23 +14,38 @@ test -n "$ASDF_INSTALL_PATH" || {
 }
 
 _get_arch() {
+  local arch; arch=$(uname -m)
+  case $arch in
+    armv*) arch="arm";;
+    aarch64) arch="arm64";;
+    x86) arch="386";;
+    x86_64) arch="amd64";;
+    i686) arch="386";;
+    i386) arch="386";;
+  esac
+  echo "$arch"
+}
+
+_get_platform() {
 	uname | tr '[:upper:]' '[:lower:]'
 }
 
 _get_download_url() {
 	local -r version="$1"
 	local -r platform="$2"
+	local -r arch="$3"
 
-	echo "https://github.com/mvdan/sh/releases/download/v${version}/shfmt_v${version}_${platform}_amd64"
+	echo "https://github.com/mvdan/sh/releases/download/v${version}/shfmt_v${version}_${platform}_${arch}"
 }
 
 install() {
-	local -r install_type=$1
+	# local -r install_type=$1 #unused variable
```

Comment:
nice.. could you remove this line and fix the install arguments as well? 

like this: 
```
install "$ASDF_INSTALL_VERSION" "$ASDF_INSTALL_PATH"
```